package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-config-formidable",
   "description": "A set of default eslint configurations from formidable",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "authors": "eric.baer@formidable.com, ryan.roemer@formidable.com, kyle.cesmat@formidable.com",
```

Comment:
I'd say that _mostly_ looks good, but generally speaking I never tag versions of any kinds in PRs, opting instead for a "version commit/merge" for the publishing event. For instance, here that could be done with:

```sh
$ npm version 1.0.0
```

(Assuming there's no safeguard in `npm` that prohibits doing a version publish at the same version when manually specified).

_or_, just as easy, we revert and for **this release only** skip `npm version` because it's already at 1.0.0, which should be our first publish.